components/date-time/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { DatePicker } from './date';
+import { TimePicker } from './time';
+
+export { DatePicker } from './date';
+export { TimePicker } from './time';
+
+export function DateTimePicker( { currentDate, onChange, locale, ...args } ) {
```

Comment:
Maybe this could be exported as "default"

Reply:
Exporting the function as default doesn't seem possible; doing so returns undefined when React attempts to render it.